src/IB/IBFEMethod.cpp

```diff
@@ -18,6 +18,7 @@
 #include "ibamr/IBFEDirectForcingKinematics.h"
 #include "ibamr/IBFEMethod.h"
 #include "ibamr/IBHierarchyIntegrator.h"
+#include "ibamr/ibamr_enums.h"
```

Comment:
For better or worse, `clangd` knows how to use `IWYU`. :-D

Reply:
That's good because it sure doesn't work with GCC!